_integration-test/run.py

```diff
@@ -0,0 +1,176 @@
+import subprocess
+import os
+from functools import lru_cache
+import requests
+import pytest
+import sentry_sdk
+import time
+import json
+
+SENTRY_CONFIG_PY = "sentry/sentry.conf.py"
+SENTRY_TEST_HOST = os.getenv("SENTRY_TEST_HOST", "http://localhost:9000")
+TEST_USER = "test@example.com"
+TEST_PASS = "test123TEST"
+TIMEOUT_SECONDS = 60
+
+
+def run_shell_command(cmd, **kwargs) -> subprocess.CompletedProcess:
+    return subprocess.run(cmd, shell=True, check=True, **kwargs)
+
+
+def poll_for_response(request: str, session: requests.Session) -> requests.Response:
+    for i in range(TIMEOUT_SECONDS):
+        response = session.get(request, headers={"Referer": SENTRY_TEST_HOST})
+        if response.status_code == 200:
+            break
+        time.sleep(1)
+    assert response.status_code == 200
```

Comment:
better would be to use `for: ... else: raise AssertionError('timeout waiting for ...')`

Reply:
did not know that existed! 👍